src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs

```diff
@@ -85,6 +85,33 @@ struct UserDataValue
         public string GetFieldValue() => _f;
     }
 
+    class UserDataGenericClass<T>
+    {
+        public const string StaticGenericFieldName = nameof(_GF);
+        public const string GenericFieldName = nameof(_gf);
+        public const string StaticGenericMethodName = nameof(_GM);
```

Comment:
Do you plan to add method tests to this PR too so that these strings and methods are used?

Reply:
These will be used when we enable more, not in this PR. I didn't use them right here as there is no need, but they represent the pattern. I didn't see any reason to not include them because that is what they will be. I'd prefer to leave them there instead of kicking off the CI again.